kokobot/cogs/util.py

```diff
@@ -83,3 +84,20 @@ def __str__(self):
             sent = await ctx.send('Cannot shutdown as {}'.format(ctx.message.author))
             await ctx.message.delete(delay=5)
             await sent.delete(delay=5)
+        
+    @commands.command()
+    async def ava(self, ctx):
```

Comment:
Can you add a string comment below here that tells what this command is doing?

See the other functions in this file for examples. This comment will appear on the $help command.